GSG-G7/mental-snapp
# Forgot Password

issue: #14 **Forgot Password**
## What did I do?
- I made the forgot password page 
- I made it responsive and it looks good in most  screen sizes

## How to test?
Just go to the `/forgot-password` route and you shall see the page 

**Note**: there are things which need to be done in the future like making the firebase function
